lib/block_view_types.go

```diff
@@ -668,9 +668,10 @@ const (
 	OperationTypeCoinLockupTransfer            OperationType = 46
 	OperationTypeCoinUnlock                    OperationType = 47
 	OperationTypeUpdateCoinLockupParams        OperationType = 48
-	OperationTypeStakeDistribution             OperationType = 49
+	OperationTypeStakeDistributionRestake      OperationType = 49
 	OperationTypeSetValidatorLastActiveAtEpoch OperationType = 50
-	// NEXT_TAG = 51
+	OperationTypeStakeDistributionPayToBalance OperationType = 51
```

Comment:
May as well give this a value of 50 since we're renaming the operation types
```suggestion
	OperationTypeStakeDistributionPayToBalance OperationType = 50
```